### PR TITLE
Handle file renames in diffs

### PR DESCRIPTION
**Pull request by makhov**
Issue #72

### CODE REVIEW

File: models/git_diff.go [hunk 1]
```diff
@@ -57,11 +58,13 @@ type DiffSection struct {
 
 type DiffFile struct {
 	Name               string
+	OldName   string
```
Comment (tboerger) [inline models/git_diff.go]: Wrong indent

File: models/git_diff.go [hunk 2]
```diff
@@ -57,11 +58,13 @@ type DiffSection struct {
...
 	Index              int
 	Addition, Deletion int
 	Type               int
 	IsCreated          bool
 	IsDeleted          bool
+	IsRenamed bool
```
Comment (tboerger) [inline models/git_diff.go]: Wrong indent

**Comment by tboerger** [comment]
Small NITPICK, beside that LGTM

**Comment by xPaw** [comment]
Is it only considering renames when files match 100%? GitHub does it in a smarter way, even if there are some changes in the file, but they are mostly the same, it is still being recognized as a rename.

**Comment by makhov** [comment]
@xPaw Thanks, fix it.

**Comment by xPaw** [comment]
What?

**Comment by makhov** [comment]
I removed 100% file matching at #86.

**Comment by xPaw** [comment]
Ah right, cool!

**Comment by xPaw** [comment]
Also, it's not showing old file name in diff stats.

**Pull request merged**